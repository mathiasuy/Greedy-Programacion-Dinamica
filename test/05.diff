1c1
< 1>Cantidad: 45435
---
> 1>Cantidad: 0
