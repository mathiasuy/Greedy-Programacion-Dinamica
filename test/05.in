test_tiempo
10000000-45435-1-20-10
Fin
