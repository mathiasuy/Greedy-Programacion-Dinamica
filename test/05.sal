1>Cantidad: 0
2>Fin.
