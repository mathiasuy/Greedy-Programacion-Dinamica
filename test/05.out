1>Cantidad: 45435
2>Fin.
